handler/api/src/main/java/io/knotx/fragments/handler/api/actionlog/ActionLog.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.fragments.handler.api.actionlog;
+
+import static com.google.common.collect.ImmutableList.copyOf;
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.StreamSupport;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class ActionLog {
```

Comment:
This class is used in the `FragmentResult` class. The `FragmentResult` class is serialised to JSON when communicating via the Vert.x event bus. I think this class should be annotated with `@DataObject`.